intranet/static/js/eighth/distribute_group.js

```diff
@@ -37,49 +38,49 @@ $(function() {
     });
 
     distribute = function() {
-        var acts = [];
-        var act_names = []
+        var acts = [],
+            act_names = [];
         var $salls = $(".select-all");
+
         $salls.each(function() {
             acts.push($(this).attr("data-name"));
             act_names.push($(this).parent().text());
         });
 
         var $rows = $("tr.user-row");
-        var max = $rows.length / $salls.length;
-        max = Math.floor(max);
-        console.debug("max:",max);
-        var rowi = 0;
-        var acti = 0;
-        var curi = 0;
-        var done = 0;
+        var max = Math.floor($rows.length / $salls.length);
+        console.debug("max:", max);
+        var rowi = 0,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["rowi" is defined but never used](https://www.codacy.com/app/tjhsst-ion/ion/file/2755452138/issues/source?bid=3212528&fileBranchId=3355681#l53)